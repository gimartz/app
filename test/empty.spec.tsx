import * as React from 'react';
import { render } from '@testing-library/react';
import { ImageListType } from '../src';
import App from './fixtures/app';

describe('testing empty case', () => {
  it('should render component', () => {
    const { container } = render(<App value={[] as ImageListType} />);
    expect(container).toMatchSnapshot();
  });
});
