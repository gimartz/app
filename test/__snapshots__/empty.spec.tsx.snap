// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing empty case should render component 1`] = `
<div>
  <div
    id="app"
  >
    <input
      accept="image/*"
      style="display: none;"
      type="file"
    />
    <div
      class="upload__image-wrapper"
    >
      <button
        id="btn-upload"
        type="button"
      >
        Click or Drop here
      </button>
      <button
        id="btn-remove"
        type="button"
      >
        Remove all images
      </button>
      <div
        id="list-images"
      />
    </div>
  </div>
</div>
`;
